docs/developer-guide/how-to/how-to-use-tokio-console.md

```diff
@@ -0,0 +1,24 @@
+# How to use tokio-console in GreptimeDB
+
+This document introduces how to use the [tokio-console](https://github.com/tokio-rs/console) in GreptimeDB.
+
+First, build GreptimeDB with feature `common-telemetry/console`. Also the `tokio_unstable` cfg must be enabled:
+
+```bash
+RUSTFLAGS="--cfg tokio_unstable" cargo build -F cmd/tokio-console
```

Comment:
does the  flag `-F cmd/tokio-console` mean the feature `common-telemetry/console`